### PR TITLE
Added section "Files to ship with the app"

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

File: microsoft-edge/webview2/concepts/distribution.md [hunk 1]
```diff
@@ -268,6 +268,31 @@ Installing the Fixed Version of the WebView2 Runtime on the client causes [Micro
         Permission for PlayReady  
     :::image-end:::  
 
+### Files to ship with the app
+
+The `WebView2Loader` code needs to be shipped with the app. This can be done by [statically linking]() `WebView2Loader.lib` into the app binaries, or by including the `WebView2Loader.dll` that matches the app's architecture. 
+
+Note that the `WebView2Loader.dll` is a native and architecture-specific binary, and so you need to include all flavors that you expect your app to run in. For example, for x86 you would include the x86 `WebView2Loader.dll`, and for AnyCPU you would include the x86, x64, and arm64 `WebView2Loader.dll` files. These are loaded from specific architecture folders.
+
+Example folder structure:
```
Comment (champnic) [inline microsoft-edge/webview2/concepts/distribution.md:277]: "Example native app folder structure:"

File: microsoft-edge/webview2/concepts/distribution.md [hunk 2]
```diff
@@ -268,6 +268,31 @@ Installing the Fixed Version of the WebView2 Runtime on the client causes [Micro
...
+```
+\<myApp>
+    \WebView2Loader.dll
+```
+
+For .NET apps, you also need to include the WebView2 .NET assemblies for the core WebView2 functionality (`Microsoft.Web.WebView2.Core.dll`) and for the WPF/WinForms-specific functionality (`Microsoft.Web.WebView2.Winforms.dll` or `Microsoft.Web.WebView2.WPF.dll`).
```
Comment (champnic) [inline microsoft-edge/webview2/concepts/distribution.md:283]: "For .NET managed apps,"

File: microsoft-edge/webview2/concepts/distribution.md [hunk 3]
```diff
@@ -268,6 +268,31 @@ Installing the Fixed Version of the WebView2 Runtime on the client causes [Micro
         Permission for PlayReady  
     :::image-end:::  
 
+### Files to ship with the app
+
+The `WebView2Loader` code needs to be shipped with the app. This can be done by [statically linking]() `WebView2Loader.lib` into the app binaries, or by including the `WebView2Loader.dll` that matches the app's architecture. 
+
+Note that the `WebView2Loader.dll` is a native and architecture-specific binary, and so you need to include all flavors that you expect your app to run in. For example, for x86 you would include the x86 `WebView2Loader.dll`, and for AnyCPU you would include the x86, x64, and arm64 `WebView2Loader.dll` files. These are loaded from specific architecture folders.
```
Comment (champnic) [inline microsoft-edge/webview2/concepts/distribution.md:275]: "For managed apps these are loaded..."

**Review by champnic — CHANGES_REQUESTED** [review]

**Comment by jasonstephen15** [comment]
@mikehoffms - ready to merge

File: microsoft-edge/webview2/concepts/distribution.md [hunk 4]
```diff
@@ -268,6 +268,31 @@ Installing the Fixed Version of the WebView2 Runtime on the client causes [Micro
         Permission for PlayReady  
     :::image-end:::  
 
+### Files to ship with the app
+
+The `WebView2Loader` code needs to be shipped with the app. This can be done by [statically linking]() `WebView2Loader.lib` into the app binaries, or by including the `WebView2Loader.dll` that matches the app's architecture.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/distribution.md:273]: todo: i'll try to fix link

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**